RELEASE/scripts/autoscend/quests/level_11.ash

```diff
@@ -2355,9 +2355,22 @@ boolean L11_redZeppelin()
 	{
 		if(cloversAvailable() >= 3 && auto_shouldUseWishes())
 		{
-			makeGenieWish($effect[Fifty Ways to Bereave Your Lover]); // +100 sleaze dmg
-			makeGenieWish($effect[Dirty Pear]); // double sleaze dmg
-		}
+			if (auto_haveGenieBottle()) {
```

Comment:
To make it easier to add wishing other effects in the future, it would be nice to have a new function `auto_wishForEffect()` or similar. Would do what you do here - try getting via genie and if that fails try the monkey paw

Reply:
This is done.